src/NationalInstruments.Analyzers/Properties/Resources.resx

```diff
@@ -414,6 +414,16 @@
   <data name="NI1816X_Title" xml:space="preserve">
     <value>Disposable should not have finalizer</value>
   </data>
+  <data name="NI1019_Description" xml:space="preserve">
+    <value>The default implicit equality method implementation for records is to compare enumerable types by reference, not the element values;  thus it is important for records with enumerable properties to explicitly implement equality and compare the enumerables using SequenceEqual or UnorderedEquals (depending on whether ordering matters).</value>
```

Comment:
"Default implicit" seems redundant.

What do you think of the following?

"By default, record equality is determined by comparing property values. If one of those properties is an enumerable, though, its "value" is a reference to an object. Thus, you must explicitly implement Equals(TRecord other) and compare enumerable elements with SequenceEqual (if order matters) or UnorderedEquals for the comparison to make logical sense."

Reply:
I think it's important to highlight that Enumerable.Equals compares the enumerable reference equality, and why that is bad.

I modified it slightly to this:

"By default, record equality is determined by calling Equals on property values. An enumerable's Equals method compares the enumerable's reference, making it a poor choice for determining whether two enumerables have the same elements . Thus, you must explicitly implement Equals(TRecord other) and compare enumerable elements with SequenceEqual (if order matters) or UnorderedEquals for the comparison to make logical sense."

What do you think?

Reply:
"poor choice" is a bit of an understatement, haha, but otherwise, I can live with it ;)